Objects/genericaliasobject.c

```diff
@@ -11,8 +11,16 @@ typedef struct {
     PyObject *args;
     PyObject *parameters;
     PyObject* weakreflist;
+    // Whether we're a starred type, e.g. *tuple[int].
+    // Only supported for `tuple`.
+    int starred;
```

Comment:
I plan on reviewing the C code once you backport the change of using `bool`.

Reply:
Oops, sorry. Fixed.